internal/db/p2p_replicator.go

```diff
@@ -23,9 +25,15 @@ import (
 	"github.com/sourcenetwork/defradb/errors"
 	"github.com/sourcenetwork/defradb/event"
 	"github.com/sourcenetwork/defradb/internal/core"
+	coreblock "github.com/sourcenetwork/defradb/internal/core/block"
 	"github.com/sourcenetwork/defradb/internal/merkle/clock"
 )
 
+const (
+	retryLoopInterval = 2 * time.Second
+	retryTimeout      = 10 * time.Second
```

Comment:
todo: Without chasing down the usage of these properties it is quite hard to guess their difference.  Please add some documentation to them.

Reply:
✅